libazureinit/src/imds.rs

```diff
@@ -133,71 +123,78 @@ mod tests {
                 "adminUsername": "MinProvAgentUser",
                 "computerName": "AzTux-MinProvAgent-Test-0001",
                 "disablePasswordAuthentication": "true"
-              }
+              },
+              "publicKeys": [
+                {
+                  "keyData": "ssh-rsa test_key1",
+                  "path": "/path/to/.ssh/authorized_keys"
+                },
+                {
+                    "keyData": "ssh-rsa test_key2",
+                    "path": "/path/to/.ssh/authorized_keys"
```

Comment:
Can you please shift the 2 lines, both keyData and path, by 2 bytes, so that the indent can be aligned with `test_key1`?